setup.py

```diff
@@ -1,10 +1,27 @@
 #!/usr/bin/env python
 from setuptools import find_packages, setup
+from setuptools.command.build_ext import build_ext
 
 long_description = (
     open("README.rst").read() + '\n\n' + open("HISTORY.rst").read()
 )
 
+class BuildClibBeforeExt(build_ext):
```

Comment:
I don't mind, but if you want to keep this approach then can we add a comment above it to explain that we need this because `build_ext` is an old-style class in Python 2?